example/app.js

```diff
@@ -67,6 +69,10 @@ class App extends React.Component {
     })
   }
 
+  setInitialValues(initialValues) {
```

Comment:
Given what this does, let's remove "initial" from this function and values. It just sets the `values` variable in the state, so this more accurately reflects what it does:
```
setValues(values) {
  this.setState({ values });
}
```